samples/net/zperf/overlay-802154-subg.conf

```diff
@@ -0,0 +1,20 @@
+CONFIG_BT=n
+
+# Disable IPv4
+CONFIG_NET_IPV4=n
+
+CONFIG_NET_CONFIG_NEED_IPV6=y
+CONFIG_NET_CONFIG_NEED_IPV4=n
+CONFIG_NET_CONFIG_MY_IPV4_ADDR=""
+CONFIG_NET_CONFIG_PEER_IPV4_ADDR=""
+
+CONFIG_NET_L2_IEEE802154=y
+CONFIG_NET_L2_IEEE802154_SHELL=y
+CONFIG_NET_L2_IEEE802154_LOG_LEVEL_INF=y
+CONFIG_NET_L2_IEEE802154_FRAGMENT_REASS_CACHE_SIZE=32
+CONFIG_IEEE802154_CC13XX_CC26XX_SUB_GHZ_NUM_RX_BUF=6
```

Comment:
This is platform-specific config, it shouldn't be present in a generic overlay file. It should end up in platform overlay in `boards` subdirectory.